### DISCUSSION TITLE
Incoming sessions are recorded in the "recent sessions"

### DISCUSSION THREAD

**Discussion by agamenabar** [rustdesk/rustdesk#4866] [Feature Request]
In the latest version 1.2.0, outgoing and incoming sessions are recorded in the "recent sessions" screen. This is a problem for maintenance, as several clients could click on the last incoming connection back.
You could put a checkbox in the configuration to disable recording of incoming sessions.

**Comment by 21pages** [comment]
related to https://github.com/rustdesk/rustdesk/commit/b5cf9592067562a9ff4651524d186d02c880bd04

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/4873
